src/test_utils.jl

```diff
@@ -56,10 +57,13 @@ end
     is_transformable(x)
 
 Determine if `x` is both a valid input and output of any [`Transform`](@ref), i.e. that it
-follows the [`transform`](@ref) interface.
-Currently, all subtypes of `Table`s and `AbstractArray`s are transformable.
+has an `apply` method defined and therefore follows the [`transform`](@ref) interface.
 """
-is_transformable(::AbstractArray) = true
-is_transformable(x) = Tables.istable(x)
+function is_transformable(T::Type)
+    Tables.istable(T) && return true  # cannot directly check against the method with no type
+    return !isempty(methodswith(T, FeatureTransforms.apply; supertypes=true))
```

Comment:
as @bencottier also pointed out in #74 we can't use `hasmethod` since it would return `true` for any type due to the method we wrote for Tables.

Reply:
`methodswith` doesn't account for the position of `T` within the `apply` arguments so this function claims that all `Transforms` are also transformable.

Reply:
hmm...is that weird? I could amend it so that `is_transformable(::Transform) == false`?

Reply:
> I could amend it so that is_transformable(::Transform) == false?

That seems more correct.  Otherwise it would seem like you could do something like `FeatureTransforms.apply(Power(2), Power(3))` which is incorrect. I'm not sure if there's a situation in which a `Transform` might be transformable though.

Reply:
yeah that makes sense - I've changed it now